beebop/app.py

```diff
@@ -0,0 +1,14 @@
+from flask import Flask
+
+from beebop import __version__, versions
+
+app = Flask(__name__)
+
+
+@app.route('/version')
+def report_version():
+    """
+    report version of beebop (and poppunk,ska in the future)
+    """
+    vers_json=versions.get_version(["beebop"])
```

Comment:
There should be spaces around `=` :)

More problematically the lint action is not working as expected, it's reporting

```
./tests/test_beebop.py:16:47: E225 missing whitespace around operator
./tests/test_beebop.py:17:1: W391 blank line at end of file
./beebop/app.py:3:1: F401 'beebop.__version__' imported but unused
./beebop/app.py:[13](https://github.com/bacpop/beebop_py/runs/5305053510?check_suite_focus=true#step:5:13):[14](https://github.com/bacpop/beebop_py/runs/5305053510?check_suite_focus=true#step:5:14): E225 missing whitespace around operator
./beebop/addition.py:9:14: W292 no newline at end of file
2     E225 missing whitespace around operator
1     F401 'beebop.__version__' imported but unused
1     W292 no newline at end of file
1     W391 blank line at end of file
```

but not failing. I get similar results out of pycodestyle

```
rich@wpia-dide136:~/Documents/Projects/bacpop/beebop_py$ pycodestyle .
./beebop/addition.py:9:14: W292 no newline at end of file
./beebop/app.py:13:14: E225 missing whitespace around operator
./beebop/versions.py:19:80: E501 line too long (87 > 79 characters)
./tests/test_beebop.py:16:47: E225 missing whitespace around operator
./tests/test_beebop.py:16:80: E501 line too long (96 > 79 characters)
./tests/test_beebop.py:17:1: W391 blank line at end of file
```

I suggest removing the flake8 step from the main action and put it in a second linting action, then verify that it fails, then de-linting